habitat/datasets/eqa/mp3d_eqa_dataset.py

```diff
@@ -57,6 +58,14 @@ def __init__(self, config: Config = None) -> None:
         with gzip.open(config.DATA_PATH.format(split=config.SPLIT), "rt") as f:
             self.from_json(f.read(), scenes_dir=config.SCENES_DIR)
 
+        if ALL_SCENES_MASK not in config.CONTENT_SCENES:
+            scenes_to_load = set(config.CONTENT_SCENES)
+            self.episodes = [
```

Comment:
As I understand this part of code filters episode form scenes that were not specified, but loaded in main dataset file. Maybe, use `dataset.filter` here.